<launch>
    <node pkg="ryuw122_tools" type="at_sender.py" name="ryuw122_at" output="screen">
        <param name="port" value="/dev/ttyTHS1"/>
        <param name="baud" value="115200"/>
        <param name="timeout" value="1.0"/>
        <param name="read_period" value="0.05"/>
        <param name="send_period" value="1.0"/>
        <param name="send_command" value="AT+ANCHOR_SEND=ANTENA01,4,TEST"/>
    </node>
</launch>
